build/buildBuildImageBases.sh

```diff
@@ -10,72 +10,62 @@
 # - Yarn package cache
 #
 
-set -e
+set -ex
 
 declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
 
 # Load all variables
 source $REPO_DIR/build/__variables.sh
 
-IMAGE_DIR_TO_BUILD=$1
-IMAGE_TAG="${BUILD_NUMBER:-latest}"
-BUILD_IMAGE_PREFIX="$__REPO_DIR/images/build"
+# Example: dontetcore, python
+IMAGE_DIR_TO_BUILD="$1"
+UNIQUE_TAG=""
```

Comment:
nit: should we renamed this to `UNIQUE_TAG_SUFFIX`?

Reply:
I will have one more PR in this area where I will fix it (just want to avoid a whole new build for this change).